setting_up_firely_server/configuration/db_memory.rst

```diff
@@ -24,6 +24,11 @@ Using the In-Memory storage
 
 Using the In-Memory storage for the Administration API database
 ---------------------------------------------------------------
+
+
+.. warning::
+    The current version of Firely Server (v5.5.0) does not support the use of In-Memory storage as administration database for STU3 resources. This will be fixed in future versions of Firely Server. For now, it is adviced to use the SQLite backend instead.
```

Comment:
Although the STU3 is mentioned in ticket https://firely.atlassian.net/browse/VONK-5908, I believe it affects all FHIR versions. So, I suggest to remove the version in the warning. :)

```suggestion
    The current version of Firely Server (v5.5.0) does not support the use of In-Memory storage as administration database. This will be fixed in future versions of Firely Server. For now, it is adviced to use the SQLite backend instead.
```